pvlib/data/sam-library-cec-modules-2019-03-05.csv

```diff
@@ -4527,6 +4527,7 @@ Eterbright Solar Corp. Cigs-3150E1,Thin Film,0,318.600000,295.100000,2.347000,1.
 Eterbright Solar Corp. Cigs-3200A1,Thin Film,0,319.780000,299.900000,2.347000,1.9,1.235,110,6.120000,75.600000,5.420000,59,0.000612,-0.207900,45.500000,2.481587,6.198086,3.298782e-13,1.636608,128.269455,-21.742449,-0.285000,N,SAM 2018.11.11 r2,1/3/2019
 Eterbright Solar Corp. Cigs-3200E1,Thin Film,0,320.220000,299.900000,2.347000,1.9,1.235,110,6.040000,75.600000,5.400000,59.300000,0.000604,-0.207900,45.500000,2.476407,6.104666,3.087447e-13,1.587451,148.272751,-22.121744,-0.285000,N,SAM 2018.11.11 r2,1/3/2019
 Eterbright Solar Corp. Cigs-3250A1,Thin Film,0,324.964000,304.600000,2.347000,1.9,1.235,110,6.150000,75.700000,5.480000,59.300000,0.000615,-0.208175,45.500000,2.482062,6.219944,3.224702e-13,1.579775,138.905914,-21.943617,-0.285000,N,SAM 2018.11.11 r2,1/3/2019
+Example Module,Multi-c-Si,0,72,48.9,0.67,1.58,0.8,18,7.5,10.4,6.6,8.4,0.003,-0.04,65,0.473,7.545,1.94E-09,0.094,15.72,10.6,-0.5,Y,MM105,10/16/2019
```

Comment:
Rather than inserting this line, I would prefer that we change to a module from the current CEC file for this  [illustration](https://pvlib-python.readthedocs.io/en/latest/pvsystem.html#pvsystem-attributes). The Canadian Solar CS5P-220M module is often used in pvlib for testing.

Reply:
Yeah I agree. I'll make the update.